tests/support/peerManager.ts

```diff
@@ -184,25 +186,44 @@ export class RadiclePeer {
     });
   }
 
-  public async startHttpd(port: number) {
+  public async startHttpd(port?: number) {
+    if (!port) {
+      port = await getPort();
+    }
+    this.#httpdBaseUrl = {
+      hostname: "0.0.0.0",
```

Comment:
Shouldn’t this be `127.0.0.1`?

Reply:
AFAIK we usually define the UI with localhost or 127.0.0.1 but the httpd we usually bind to 0.0.0.0
We also use 0.0.0.0 as default in heartwood, so probably better to use 0.0.0.0 here.
https://app.radicle.xyz/seeds/seed.radicle.xyz/rad:z3gqcJUoA1n9HaHKufZs5FCSGazv5/tree/master/radicle-httpd/src/main.rs#L76

Reply:
> We also use 0.0.0.0 as default in heartwood, so probably better to use 0.0.0.0 here.
> https://app.radicle.xyz/seeds/seed.radicle.xyz/rad:z3gqcJUoA1n9HaHKufZs5FCSGazv5/tree/master/radicle-httpd/src/main.rs#L76

In general, the IP address 0.0.0.0 is [only used as a source address](https://en.wikipedia.org/wiki/0.0.0.0).  Here, you’re using it as a destination address. While this works with the browser network stack and also a lot of other tools it’s very unusual to see this and not standard.

In the code you linked to, the address 0.0.0.0 tells the networking stack that it should bind to all network interfaces on the host. This includes the IP address 127.0.0.1 so it is safe (and the standard) to use that as the destination address.

Reply:
Thanks @geigerzaehler for the explainer, will change it!

Reply:
done